db/migrate/20220326112159_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :Author, foregin_key: { to_table: 'users' }, index:true
```

Comment:
- [ ] Kindly note that it's always a good practice to name all columns in a Postgres table in lowercase and separate words with `_`. example `posts_pounter` instead of `PostsCounter`, `author` instead of `Author` .... Kindly rename these table's column accordingly.